inlong-manager/manager-web/sql/changes-1.8.0.sql

```diff
@@ -84,6 +84,19 @@ UPDATE inlong_cluster SET ext_params = replace(ext_params, '"tenant"', '"pulsarT
 
 ALTER TABLE `inlong_stream` MODIFY COLUMN `name` varchar(256) DEFAULT NULL COMMENT 'The name of the inlong stream page display, can be Chinese';
 
+
+CREATE TABLE IF NOT EXISTS `audit_query_source_config`
```

Comment:
It is recommended to add two fields createTime and updateTime to record the time of writing and updating